clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -394,7 +398,7 @@ public void testRename_existingFileToExistingDirName() throws ApiException, IOEx
         ObjectLocation fileObjLoc = fs.pathToObjectLocation(fileInDstDir);
         Path dst = new Path("lakefs://repo/main/existing-dir2");
         ObjectLocation dstObjLoc = fs.pathToObjectLocation(dst);
-        mockExistingDirPath(dstObjLoc, fileObjLoc);
+        mockExistingDirPath(dstObjLoc, new ArrayList<ObjectLocation>(Arrays.asList(fileObjLoc)));
```

Comment:
```suggestion
        mockExistingDirPath(dstObjLoc, ImmutableList.of(fileObjLoc));
```
(Also below, I think we can get by without `Arrays` and directly generate a list).

Alternatively, pass an array and iterate over _that_.  Mixing Lists and native arrays is too verbose if we don't have to do it.